cpp/oneapi/dal/algo/kmeans/backend/gpu/train_kernel_lloyd_dense_dpc_test.cpp

```diff
@@ -0,0 +1,97 @@
+/*******************************************************************************
+* Copyright 2020 Intel Corporation
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*******************************************************************************/
+
+#include <CL/sycl.hpp>
+
+#include "gtest/gtest.h"
+#define ONEAPI_DAL_DATA_PARALLEL
+#include "oneapi/dal/algo/kmeans.hpp"
+#include "oneapi/dal/data/accessor.hpp"
+#include "oneapi/dal/data/table.hpp"
+
+using namespace oneapi::dal;
+
+TEST(train_kernel_lloyd_dense, test1) {
+    auto selector = sycl::gpu_selector();
+    auto queue    = sycl::queue(selector);
+
+    constexpr std::int64_t row_count     = 8;
+    constexpr std::int64_t column_count  = 2;
+    constexpr std::int64_t cluster_count = 2;
+
+    const float data_host[] = { 1.0,  1.0,  2.0,  2.0,  1.0,  2.0,  2.0,  1.0,
+                                -1.0, -1.0, -1.0, -2.0, -2.0, -1.0, -2.0, -2.0 };
+    auto data               = sycl::malloc_shared<float>(row_count * column_count, queue);
+    queue.memcpy(data, data_host, sizeof(float) * row_count * column_count).wait();
+    const auto data_table = homogen_table{ row_count, column_count, data };
```

Comment:
queue

Reply:
Why resolved?

Reply:
Please follow every resolved comment with explanation - seems like you just hide the problems.